subaccounts.go

```diff
@@ -4,34 +4,39 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+
+	"github.com/pkg/errors"
 )
 
 // https://www.sparkpost.com/api#/reference/subaccounts
 var SubaccountsPathFormat = "/api/v%d/subaccounts"
-var availableGrants = []string{
+var SubaccountGrants = []string{
 	"smtp/inject",
 	"sending_domains/manage",
 	"message_events/view",
 	"suppression_lists/manage",
+	"tracking_domains/view",
+	"tracking_domains/manage",
 	"transmissions/view",
 	"transmissions/modify",
 }
-var validStatuses = []string{
+var SubaccountStatuses = []string{
 	"active",
 	"suspended",
 	"terminated",
 }
 
 // Subaccount is the JSON structure accepted by and returned from the SparkPost Subaccounts API.
 type Subaccount struct {
-	ID               int      `json:"subaccount_id,omitempty"`
```

Comment:
[Docs](https://developers.sparkpost.com/api/subaccounts.html#subaccounts-subaccounts-collection-get) and a live test contradict this json tag.